data/warehousing.json

```
@@ -0,0 +1,2731 @@
+            "default": "Order"
+          },
+          "messageFunction": {
+            "type": "string",
+            "description": "Type of order.",
+            "examples": [
+              "For salesorder/customer order = Order",
+              "For purchaseOrder = PurchaseOrder"
+            ]
+          },
+          "profile": {
+            "type": "string",
+            "description": "Profile.",
+            "default": "BringXML"
+          },
+          "messageId": {
+            "type": "string",
+            "description": "Unique id for this message.",
+            "examples": [
+              "3dcc8b71-2bf7-4574-813d-9295bd5db8c0"
+            ]
+          },
+          "updateIndicator": {
+            "type": "string",
+            "description": "Update Indicator",
+            "examples": [
+              "updateIndicator=Deletion"
+            ]
+          },
+          "testIndicator": {
+            "type": "string",
+            "description": "Whether this is a test message.",
+            "examples": [
+              "testIndicator=NO"
+            ]
+          },
+          "MessageDetails": {
+            "type": "array",
+            "description": "Message Details",
+            "items": {
+              "$ref": "#/components/schemas/MessageDetails"
+            }
+          },
+          "SupplyStructure": {
+            "type": "array",
+            "description": "Supply Structure",
+            "items": {
+              "$ref": "#/components/schemas/SupplyStructureItemData"
+            }
+          }
+        },
+        "description": "List of consignments"
+      },
+      "MessageDetails": {
+        "required": [
+          "MessageDate",
+          "MessageRecipient",
+          "MessageSender"
+        ],
+        "type": "array",
+        "properties": {
+          "MessageDate": {
+            "type": "datetime",
+            "description": "The date on which the message was sent.",
+            "examples": [
+              "<MessageDate>2021-02-12T13:38:08</MessageDate>"
+            ]
+          },
+          "MessageRecipient": {
+            "type": "array",
+            "description": "Message Recipient",
+            "items": {
+              "$ref": "#/components/schemas/MessageDetailsIds"
+            }
+          },
+          "MessageSender": {
+            "type": "array",
+            "description": "Message Recipient",
+            "items": {
+              "$ref": "#/components/schemas/MessageDetailsIds"
+            }
+          }
+        },
+        "description": "Message Details"
+      },
+      "MessageDetailsIds": {
+        "required": [
+          "partyId"
+        ],
+        "type": "string",
+        "properties": {
+          "partyId": {
+            "type": "string",
+            "description": "Customer number (6 or 11 digits) provided by Bring.",
+            "examples": [
+              "<MessageRecipient partyId=20001234567 idType=14 />"
+            ]
+          },
```

Ref. the other comment, I think the syntax would be something like:
  
  ```suggestion
            "partyId": {
              "type": "string",
              "description": "Customer number (6 or 11 digits) provided by Bring.",
              "examples": [
                "20001234567"
              ],
              "xml": {
                "attribute": true
              }
            },
  ```

---

Used Address as reference: https://api.bring.com/address/api-docs/

---

Right.. I could try that instead!